CWE.Data/DataAccessLayer.cs

```diff
@@ -330,6 +363,50 @@ public async Task CreateRank(ulong id)
         }
 
         /// <summary>
+        /// Gets all the infractions for a specified user.
+        /// </summary>
+        /// <param name="userId">The user to get the infractions for.</param>
+        /// <returns>A <see cref="IReadOnlyCollection{T}"/> containing all the infractions.</returns>
+        public async Task<IReadOnlyCollection<Infraction>> GetUserInfractions(ulong userId)
+        {
+            var infracs = this.dbContext.Infractions.Where(x => x.UserId == userId);
+            return await infracs.ToListAsync();
+        }
```

Comment:
```suggestion
        public async Task<IReadOnlyCollection<Infraction>> GetInfractions(ulong userId)
        {
            return await this.dbContext.Infractions.Where(x => x.UserId == userId).ToListAsync();
        }
```
Can be moved into one line and changed the name as there is no `GetInfractions()` (more consistent naming).